Update MyFaces to 4.1.1

volosied opened the pull request:
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

fixes #31095

volosied committed:
Update API to 4.1.1

volosied committed:
Update Impl to 4.1.1

volosied committed:
Update third party to 4.1.1 & cnf files

volosied commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=73f9646d-3cda-457a-b288-cea2108ff6f6) - :x: completed with errors/failures.)
fat.test.mode=full
fat.buckets.to.run=com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.3_fat.2,io.openliberty.org.apache.myfaces.4.0_fat,io.openliberty.org.apache.myfaces.4.1_fat

_Note: Target locations of links might be accessible only to IBM employees._

pnicolucci commented on dev/io.openliberty.org.apache.myfaces.4.1/build.gradle:
```diff
@@ -19,7 +16,7 @@ configurations {
 
 dependencies {
-   sourceJar 'org.apache.myfaces.core:myfaces-impl:4.1.0:sources'
+//    sourceJar 'org.apache.myfaces.core:myfaces-impl:4.1.0:sources'
```
Can you open an issue to get these sources uploaded or how are you handling it?

volosied replied:
https://github.com/OpenLiberty/open-liberty/issues/31096

volosied replied:
I will also message Chuck now :)

volosied commented:
!build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=3134f85b-61e4-46c7-80b9-2fd5fb73a031) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

volosied commented:
!libby ([view Open Liberty Delivery Requirements Verification](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=9a82fe11-9ce0-47f4-9442-f6b314c840e6) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

volosied committed:
Overlay 4.1.1 Source Files - MyfacesConfig & HtmlResponseStateManager

Add WebConfigParamsLogger to Faces 4.1

volosied committed:
Apply MYFACES-4721 to Faces 4.1

Update AUTOCOMPLETE_OFF_VIEW_STATE logging to one-time-code  in WebConfigParamsLogger

volosied commented:
!build  ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=71d04822-b1e0-4c22-931c-f1f4c0f7f3cf) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

volosied commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=543c2c2b-8b54-4121-a389-71965373628e) - :x: completed with errors/failures.)
spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.3_fat.2,com.ibm.ws.jsf_fat_lWAS_faces4.1,io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.security.javaeesec_fat,io.openliberty.org.apache.myfaces.4.1_fat

_Note: Target locations of links might be accessible only to IBM employees._

volosied commented:
!build 
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=dd4729ca-6b74-4dbc-8204-af25763beb9a) - :x: completed with errors/failures.)
spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.3_fat.2,com.ibm.ws.jsf_fat_lWAS_faces4.1,io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.security.javaeesec_fat,io.openliberty.org.apache.myfaces.4.1_fat

_Note: Target locations of links might be accessible only to IBM employees._

volosied committed:
Enable check_defaultLogging_AUTOCOMPLETE_OFF_VIEW_STATE in 4.1

volosied commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=6235980e-b940-4395-8ea3-f7b4845baec4) - :x: completed with errors/failures.)
spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.3_fat.2,com.ibm.ws.jsf_fat_lWAS_faces4.1,io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.security.javaeesec_fat,io.openliberty.org.apache.myfaces.4.1_fat

_Note: Target locations of links might be accessible only to IBM employees._

volosied commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=6290a505-b141-4766-82a0-c7dc80597df1) - :x: completed with errors/failures.)
spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.3_fat.2,com.ibm.ws.jsf_fat_lWAS_faces4.1,io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.security.javaeesec_fat,io.openliberty.org.apache.myfaces.4.1_fat


_Note: Target locations of links might be accessible only to IBM employees._

volosied commented:
!build
 ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=20a3a3c1-f253-4f88-abc5-1882f44ff38c) - :white_check_mark: completed successfully!)
spawn.fullfat.buckets=com.ibm.ws.jsf.2.2_fat,com.ibm.ws.jsf.2.2_fat.2,com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.3_fat.2,com.ibm.ws.jsf_fat_lWAS_faces4.1,io.openliberty.org.apache.myfaces.4.0_fat,com.ibm.ws.security.javaeesec_fat,io.openliberty.org.apache.myfaces.4.1_fat

_Note: Target locations of links might be accessible only to IBM employees._

volosied committed:
Ignore context param log message check for checkpoint tests
